.github/workflows/release-cli.yml

```diff
@@ -550,7 +550,7 @@ jobs:
 
       - name: Install on Windows using Scoop
         run: |
-          Invoke-Expression (New-Object System.Net.WebClient).DownloadString('https://get.scoop.sh')
+          iex "& {$(irm get.scoop.sh)} -RunAsAdmin"
```

Comment:
https://github.com/keboola/keboola-as-code/actions/runs/4243846586/jobs/7391100438#step:6:24
![image](https://user-images.githubusercontent.com/19371734/229432971-7bd5007d-a0c0-4ddc-b3f7-134111e84e85.png)


Fixnute podla odporucania:
https://github.com/ScoopInstaller/Install#for-admin